lib/endpoints/class-wp-rest-controller.php

```diff
@@ -250,7 +250,6 @@ public function get_collection_params() {
 				'default'            => 10,
 				'minimum'            => 1,
 				'maximum'            => 100,
-				'sanitize_callback'  => 'absint',
```

Comment:
> per_page=-1 will be changed to per_page=1 by absint().

Doesn't validation happen before this, though? If validation happens before this, then the validation callback should trigger the error before it hits the sanitization callback.